Content.Shared/Audio/Jukebox/JukeboxUi.cs

```diff
@@ -0,0 +1,18 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Audio.Jukebox;
+
+
+[Serializable, NetSerializable]
+public enum JukeboxUiKey : byte
+{
+    Key,
+}
+
+[Serializable, NetSerializable]
+public sealed class JukeboxBoundInterfaceState : BoundUserInterfaceState
+{
+    public NetEntity? Audio;
+    public ProtoId<JukeboxPrototype>? SelectedSong;
```

Comment:
These are already on the jukebox component, can the client UI use the values from that instead?